### PR TITLE
13160 BEN corrections UI adjustments

### PR DESCRIPTION
**Pull request by lambert-alex**
*Issue #:* /bcgov/entity#13160

*Description of changes:*
- Removed the highlighted box from deactivated checkbox
- the rest of the work completed for this ticket was done in [bcrs-shared-components 150](https://github.com/bcgov/bcrs-shared-components/pull/150)
- imported the the shareStructure component from the work done in the PR above

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Review by severinbeauvais — APPROVED** [review]
LGTM!

Please wait for another review before merging.

**Review by lewischenstudio — APPROVED** [review]
LGTM too

**Pull request merged**
